src/Desktop/UIAutomation/CustomObjects/Registrar.cs

```diff
@@ -59,6 +59,29 @@ public void RegisterCustomProperty(CustomProperty prop)
         public Dictionary<int, CustomProperty> GetCustomPropertyRegistrations() { return new Dictionary<int, CustomProperty>(_idToCustomPropertyMap); }
 #pragma warning restore CA1024 // Use properties where appropriate: this is a copy of the object, not the object itself
 
+        /// <summary>
+        /// Notifies ConverterRegistrationAction about all of the custom property registrations in
+        // the passed-in dictionary. This is used, for instance, when switching between file and
+        // live mode in Accessibility Insights for Windows.
```

Comment:
These lines should also be triple-slash comments
```suggestion
        /// the passed-in dictionary. This is used, for instance, when switching between file and
        /// live mode in Accessibility Insights for Windows.
```